destroy-vm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# delete and purge relativepath VM
+if ( multipass list | grep relativepath )
+then
```

Comment:
Let's change the conditional to check if it does not exists so `if ( multipass list | grep relativepath )` you can add a `!` to any command and it will flip it's exit code. So we could do something like  `if ( ! multipass list | grep relativepath )` to make it true that relativepath doesn't exist. Then you can switch what's in your `then` with what's in your `else` like we have it in other scripts.

https://bash.cyberciti.biz/guide/Logical_Not_!